app/frontend/helpers/notes_helper.js

```diff
@@ -0,0 +1,19 @@
+
+const removeMd = require('remove-markdown');
```

Comment:
Just commenting ... Good idea to use that library for this purpose ... Good job! 👍